src/Qir/Runtime/lib/QSharpCore/qsharp__core__qis.hpp

```diff
@@ -1,24 +1,11 @@
 #pragma once
 
-#include <cstdint>
-
-#include "CoreTypes.hpp"
+#include "CoreTypes.hpp"    // QUBIT, PauliId, RESULT
 
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
 struct QirArray;
-struct QirCallable;
```

Comment:
Why were these declarations removed?

Reply:
Because they are either not used below or are declared in the included header.